feat: add live preview method setting for prompt execution

ltdrdata opened the pull request:
## Summary

Add frontend setting to override live preview method per prompt execution.

## Changes

- **What**: New setting `Comfy.Execution.PreviewMethod` allows users to override preview method (default/none/auto/latent2rgb/taesd) from frontend. Applied to Queue Prompt, Queue Front, Run Selected Nodes, and Auto Queue.
- **Dependencies**: Requires backend support from comfyanonymous/ComfyUI#11261

## Review Focus

- `'default'` option does not send `preview_method` to backend (uses server CLI setting)
- Legacy UI intentionally not modified (deprecated, maintains backward compatibility)
- `versionAdded: '1.35.3'` assigned tentatively; adjust as needed for actual release version

┆Issue is synchronized with this [Notion page](https://www.notion.so/PR-7385-feat-add-live-preview-method-setting-for-prompt-execution-2c66d73d365081759c9cebaec29f451c) by [Unito](https://www.unito.io)

DrJKL commented on src/scripts/api.ts:
```diff
@@ -104,6 +110,12 @@ interface QueuePromptOptions {
+   * Values: 'default' | 'none' | 'auto' | 'latent2rgb' | 'taesd'
+   * 'default' uses the server's CLI setting and is not sent to backend.
+   */
+  previewMethod?: string
```
Same as above:
```suggestion
  previewMethod?: 'default' | 'none' | 'auto' | 'latent2rgb' | 'taesd'
```

DrJKL commented on src/scripts/app.ts:
```diff
@@ -1325,9 +1326,10 @@ export class ComfyApp {
     batchCount: number = 1,
-    queueNodeIds?: NodeExecutionId[]
+    queueNodeIds?: NodeExecutionId[],
+    previewMethod?: string
```
Also, instead of wiring this through, could we grab the setting value in here?

DrJKL commented on src/scripts/app.ts:
```diff
@@ -1325,9 +1326,10 @@ export class ComfyApp {
+    previewMethod?: string
   ): Promise<boolean> {
-    this.queueItems.push({ number, batchCount, queueNodeIds })
+    this.queueItems.push({ number, batchCount, queueNodeIds, previewMethod })
```
Like this:
```suggestion
    const previewMethod = useSettingStore().get(
          'Comfy.Execution.PreviewMethod'
    )
    this.queueItems.push({ number, batchCount, queueNodeIds, previewMethod })
```

ltdrdata committed:
feat: add live preview method setting for prompt execution

Allow users to override the preview method (default/none/auto/latent2rgb/taesd)
from frontend settings. 'default' uses the server CLI setting.

- Add PreviewMethod type using zod schema in apiSchema.ts
- Add Comfy.Execution.PreviewMethod setting with combo options
- Read preview method setting inside queuePrompt() for centralized access
- Send preview_method in extra_data when not 'default'
